definitions/variable/emissions/emissions.yaml

```
@@ -123,52 +150,53 @@
-      not accounting for negative emissions from bioenergy with CCS (BECCS) in this sector
+      not accounting for emissions removed from the atmosphere
     unit: Mt CO2/yr
 - Emissions|CO2|Energy|Supply|{Secondary Fuel Level 2}:
     description: Emissions of carbon dioxide (CO2) for production of {Secondary Fuel Level 2}
     unit: Mt CO2/yr
     tier: "{Secondary Fuel Level 2}"
 - Gross Emissions|CO2|Energy|Supply|{Secondary Fuel Level 2}:
     description: Gross emissions of carbon dioxide (CO2) for production of {Secondary Fuel Level 2},
-      not accounting for negative emissions from bioenergy with CCS (BECCS) in this sector
+      not accounting for emissions removed from the atmosphere
     unit: Mt CO2/yr
     tier: "{Secondary Fuel Level 2}"
 - Emissions|CO2|Energy|Supply|Autoproduction:
     description: Emissions of carbon dioxide (CO2) for own-use energy supply
     unit: Mt CO2/yr
 - Gross Emissions|CO2|Energy|Supply|Autoproduction:
     description: Gross emissions of carbon dioxide (CO2) for own-use energy supply,
-      not accounting for negative emissions from bioenergy with CCS (BECCS) in this sector
+      not accounting for emissions removed from the atmosphere
     unit: Mt CO2/yr
 - Emissions|CO2|Energy|Supply|Other:
     description: Emissions of carbon dioxide (CO2) for other categories of energy supply
     unit: Mt CO2/yr
 - Gross Emissions|CO2|Energy|Supply|Other:
     description: Gross emissions of carbon dioxide (CO2) for other categories of energy supply,
-      not accounting for negative emissions from bioenergy with CCS (BECCS) in these sectors
+      not accounting for emissions removed from the atmosphere
     unit: Mt CO2/yr
 
 - Emissions|{Level-2 Species}|Energy|Demand:
     description: Emissions of {Level-2 Species} from fuel combustion in industry (IPCC category 1A2),
       residential, commercial, institutional sectors and agriculture, forestry,
       fishing (AFOFI) (IPCC category 1A4a, 1A4b, 1A4c), and the transportation sector
-      (IPCC category 1A3), excluding pipeline emissions (IPCC category 1A3ei),
-      net of negative emissions incl. demand-side use of bioenergy with CCS (BECCS)
+      (IPCC category 1A3), excluding pipeline emissions (IPCC category 1A3ei).
+      This is the sum of sources and sinks, including negative emissions such as the demand-side use 
+      of bioenergy with CCS (BECCS)
     unit: "{Level-2 Species}"
 - Gross Emissions|CO2|Energy|Demand:
     description: Gross emissions of carbon dioxide (CO2) from fuel combustion in industry (IPCC category 1A2),
       residential, commercial, institutional sectors and agriculture, forestry,
       fishing (AFOFI) (IPCC category 1A4a, 1A4b, 1A4c), and the transportation sector
       (IPCC category 1A3), excluding pipeline emissions (IPCC category 1A3ei),
-      not accounting for negative emissions from bioenergy with CCS (BECCS) in these sectors
+      not accounting for emissions removed from the atmosphere
```

```suggestion
      not accounting for removals from the atmosphere
```

Alright I'll stop writing this now, but I think you get the point. You're not really removing "emissions", you're removing "carbon" so just writing "removals" might be simpler